props/jetbrains.props

```diff
@@ -0,0 +1,18 @@
+<Project>
```

Comment:
The other .props files use 2-space indentation. Would be nice if we could keep this consistent.